vite.telemetry.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from "vite";
+
+export default defineConfig({
```

Comment:
We unfortunately need a second build step because sveltekit overrides vite's `rollupOptions` which we need to specify a second entrypoint